src/SIL.LCModel/Infrastructure/Impl/RepositoryAdditions.cs

```diff
@@ -103,7 +105,7 @@ public IAnalysis GetObject(int hvo)
 			return (IAnalysis)m_everythingRepos.GetObject(hvo);
 		}
 
-		public bool TryGetObject(int hvo, out IAnalysis obj)
+		public bool TryGetObject(int hvo, out IAnalysis? obj)
```

Comment:
I suggest for these TryGet methods we put `[MaybeNullWhen(false)]` on the out variable, then we can remove the `?`, that says when false is returned obj can be null.